crm_compassion/model/demand_planning.py

```diff
@@ -73,3 +73,39 @@ class DemandPlanning(models.Model):
     weekly_demand_ids = fields.Many2many(
         'demand.weekly.demand', string='Weekly Demands'
     )
+
+    ##########################################################################
+    #                             ORM METHODS                                #
+    ##########################################################################
+    @api.model
+    def create(self, vals):
```

Comment:
I wouldn't send the message directly at creation. Add instead a `send` button in the view that would trigger the send to GMC.

You can also add a `state` field to know if a planning was sent or not (by default state would be draft, when successfully sent it goes to sent, and if an error occurs you can put state error)